### PR TITLE
Add Katello 3.12.3 release to toolbelt

### PR DESCRIPTION
**Pull request by chris1984**
@jturel here is the pr for 3.12.3

### CODE REVIEW

File: configs/katello/3.12.yaml [hunk 1]
```diff
@@ -4,9 +4,9 @@
 :prior_releases:
   :3.11.2:
     :redmine_version_id: 1038
-  :3.11.1:
```
Comment (jturel) [inline configs/katello/3.12.yaml]: I don't think there's a reason to remove 3.11.1 from priors

File: configs/katello/3.13.yaml [hunk 2]
```diff
@@ -2,8 +2,8 @@
 :project: katello
 :github_org: katello
 :prior_releases:
-  :3.12.2:
-    :redmine_version_id: 1053
+  :3.12.3:
```
Comment (jturel) [inline configs/katello/3.13.yaml]: 3.12.3 should be an additional item rather than a replacement for 3.12.2

**Comment by chris1984** [comment]
@jturel updated

**Comment by chris1984** [comment]
@ekohl can this get a merge when you get a free moment please?

**Comment by jlsherrill** [comment]
Thanks @chris1984 !

**Pull request merged**